README.md

```diff
@@ -123,6 +123,12 @@ and their default values.
 | `fullnameOverride`                 | Set resource fullname override                                                     | `""`                           |
 | `useSecretHtpasswd`                | Use htpasswd from `.Values.secrets.htpasswd`. This require helm v3.2.0 or above.   | `false`                        |
 | `secrets.htpasswd`                 | user and password list to generate htpasswd.                                       | `[]`                           |
+| `ingress.enabled`                  | Enable/Disable Ingress                                                             | `false`                        |
```

Comment:
@rblaine95 is this used? I don't see any ref in the code.

Reply:
It's used on line 1 of `templates/ingress.yaml` to enable/disable the ingress manifest
```go
{{- if .Values.ingress.enabled }}
{{- $fullName := include "verdaccio.fullname" . -}}
{{- $svcPort := .Values.service.port -}}
{{- $paths := .Values.ingress.paths -}}
[...]
```